profiles/management/commands/gen_realistic_users.py

```diff
@@ -0,0 +1,254 @@
+"""
+Generates a set of realistic users to help us test search functionality
+"""
+from datetime import datetime, timedelta
+from django.core.management import BaseCommand
+from django.contrib.auth.models import User
+from profiles.models import Employment, Education
+from courses.models import Program, Course, CourseRun
+from dashboard.models import CachedCertificate, CachedEnrollment
+from micromasters.utils import load_json_from_file
+
+
+USER_DATA_PATH = 'profiles/management/realistic_user_data.json'
+PROGRAM_DATA_PATH = 'profiles/management/realistic_program_data.json'
+FAKE_USER_USERNAME_PREFIX = 'fake.'
+FAKE_PROGRAM_DESC_PREFIX = '[FAKE] '
+CACHED_MODEL_LAST_REQUEST = datetime.now() + timedelta(days=365)
+
+
+# Util functions
+
+def first_matching_item(iterable, predicate):
+    """Returns the first item in an iterable that matches a given predicate"""
+    return next(x for x in iterable if predicate(x))
+
+
+def is_relation_field(model_obj, attr_name):
+    """
+    Checks if a given model attribute is a relation rather than a single value
+    """
+    return 'RelatedManager' in str(type(getattr(model_obj, attr_name)))
```

Comment:
I'm a little confused about this. If I'm looking at a user and I do:

```python
str(type(getattr(user, 'profile')))
```

I get `"<class 'profiles.models.Profile'>"`. Is this what this is supposed to capture? The Profile object is a different model, but `RelatedManager` doesn't appear in that result.